src/grokcore/chameleon/README.rst

```diff
@@ -203,7 +203,7 @@ and render it:
     This template knows about the following vars:
     <BLANKLINE>
       template (the template instance):
-       &lt;PageTemplateFile ...vars.cpt&gt;
+       &lt;PageTemplateFile ...vars.cpt'&gt;
```

Comment:
this `'` is what makes the test pass again? Wow.

Reply:
Yes, the repr of `PageTemplateFile` seems to have changed.